x/mint/module/abci.go

```diff
@@ -48,16 +48,19 @@ func GetEmissionPerMonth(
 		"totalSupply", totalSupply.String(),
 		"lockedSupply", lockedSupply.String(),
 	)
-	targetRewardEmissionPerUnitStakedToken,
-		err := keeper.GetTargetRewardEmissionPerUnitStakedToken(
-		params.FEmission,
-		ecosystemMintSupplyRemaining,
-		networkStaked,
-		circulatingSupply,
-		params.MaxSupply,
-	)
-	if err != nil {
-		return math.Int{}, math.LegacyDec{}, err
+	targetRewardEmissionPerUnitStakedToken := math.LegacyZeroDec()
+	if ctx.BlockHeight() > 0 {
```

Comment:
this should be greater than 1, not zero

https://github.com/allora-network/allora-chain/blob/3ced485211ecf438a7602b5996da872a545d1a98/x/mint/module/abci.go#L143